specs/brew/pin_homebrew_formula.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Pin a formula to its current version
+command: "brew pin {{formula_name}}"
+tags:
+  - homebrew
+description: Pins a version of a homebrew formula to its current version, i.e. it will not be updated when a newer version is available.
+arguments:
+  - name: formula_name
+    description: The formula to pin
+    default_value: ~
+source_url: "https://docs.brew.sh/Manpage"
+author: ifthisthenthat
```

Comment:
would you mind clarifying why the author here is `ifthisthenthat`? If the URL is your GitHub, would love to credit you directly in the author field as well :)

Reply:
I just chose that because that's my username on almost every platform except for GitHub.

Reply:
Ah gotcha--great username! Would you be comfortable if we used your GitHub username just to have it consistent between the `author` and the `author_url`?

Reply:
Thanks! That sounds good, I'll do that quickly.